app/src/store/voice/audioDevices.ts

```diff
@@ -13,17 +13,23 @@ export const getAudioDevices = createAsyncThunk<
   { audioDevices: AudioDeviceInfo[]; selectedDevice: AudioDeviceInfo | null },
   void,
   { state: State; dispatch: Dispatch }
->('voice/getAudioDevices', async () => {
-  const { audioDevices, selectedDevice } = await voice.getAudioDevices();
+>('voice/getAudioDevices', async (_, { rejectWithValue }) => {
+  try {
```

Comment:
I'm not sure this `try` catch will actually do anything at the moment. I don't believe `voice.getAudioDevices` can fail with how the native code base is currently set up, but I think this is good future-proofing. Something to note, doesn't change the review.